### ISSUE TITLE
Keeping comments when the original post was removed by the author.

### ISSUE THREAD

**Issue by leftside2** [HelloZeroNet/ZeroTalk#63]
Some users delete their posts without showing any respect to the commenters' efforts, so that no matter how many comments were attached all of them disappear too.

So people become reluctant to comment any more so that the ZeroNet community's falling down.

Allow the topic creator to choose what to remove among title, body and author ID.
In case the bulletin board doesn't allow anonymous posting, when he selects 'remove author ID' show him the msg saying 'Removing author ID will remove the title and the body together and they will not be recovered.' If click 'yes', remove all 3 items, but still remains comments under the topic.

http://127.0.0.1:43110/Me.ZeroNetwork.bit/?Profile/1SunAWK2VUT9GQK32MpwRfFPVgcBSJN9a/16dEowtNbV89NxPcDrAKkmnuXtiJVoVcDm/leftside@zeroid.bit

sos4sos@riseup.net 2018-03-07 16-43-58

**Comment by slrslr** [comment]
I do not agree. I think topic starter should have this right about his own topic.

**Comment by leftside2** [comment]
Plz refer to the discussion. [Click here.](http://127.0.0.1:43110/Talk.ZeroNetwork.bit/?Topic:1520413394_16dEowtNbV89NxPcDrAKkmnuXtiJVoVcDm)
